lib/puma/io_buffer.rb

```diff
@@ -22,6 +22,16 @@ def to_s
       read
     end
 
+    # Read & Reset - returns contents and resets
+    # @return [String] StringIO contents
+    def rd_and_rst
```

Comment:
```suggestion
    def read_and_reset
```

Reply:
Fixed, pushed back into 'Updates for response body processing'